services/application.js

```diff
@@ -363,6 +360,18 @@ function verifyReviewerPermission(context){
         .verifyRole([ROLES.ADMIN, ROLES.FEDERAL_LEAD]);
 }
 
+const toISOTime = (aApp) => {
```

Comment:
This function name is a bit misleading, people may think we are converting the object into an ISO time. I suggest rename it to something like transformDateTime, transformDateTimeFormat or similar.

Reply:
updated!